zebra-chain/src/sprout/tree.rs

```diff
@@ -84,7 +96,7 @@ pub struct Position(pub(crate) u64);
 /// each treestate.
 #[derive(Clone, Copy, Default, Eq, PartialEq, Serialize, Deserialize, Hash)]
 #[cfg_attr(any(test, feature = "proptest-impl"), derive(Arbitrary))]
-pub struct Root([u8; 32]);
+pub struct Root(pub [u8; 32]);
```

Comment:
Does this need to be public?

Reply:
I made it public because of this test https://github.com/ZcashFoundation/zebra/blob/e084748bd84f49220cfda0d4c3c85587bc186495/zebra-chain/src/sprout/tests/tree.rs#L52
I can get rid of the test if it's too trivial and remove the `pub`.

Reply:
Try ```incremental_root.into()``` as that trait is implemented for `[u8; 32]`